<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Industry</class>
 <widget class="QDialog" name="Industry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Industry{ background-image: url(:/bt/image/industry_bg.png);}</string>
  </property>
  <widget class="QLabel" name="RTemplateLable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>885</width>
     <height>901</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ToBeIdentified">
   <property name="geometry">
    <rect>
     <x>985</x>
     <y>130</y>
     <width>885</width>
     <height>901</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>0</y>
     <width>200</width>
     <height>116</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
